src/components/SearchBar.vue

```diff
@@ -6,10 +6,11 @@
       flat
       round
       icon="arrow_back"
-      :aria-label="$t('Back')"
+      :aria-label="$t('back')"
       @click="$router.back()"
     />
-
+    
+    <!-- eslint-disable vue-i18n/no-dynamic-keys -->
     <q-toolbar-title v-if="!viewSearch">{{ $t(title) }}</q-toolbar-title>
```

Comment:
```suggestion
    <q-toolbar-title v-if="!viewSearch">{{ title }}</q-toolbar-title>
```
El mateix aquí: ens haurien de passar el títol ja traduït! No pas passar-nos la clau de traducció i traduir-la aquí.